playground/frontend/lib/constants/links.dart

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const kReportIssueLink = 'https://issues.apache.org/jira/projects/BEAM/issues/';
+const kGooglePrivacyPolicyLink = 'https://policies.google.com/privacy';
```

Comment:
hm why are we using the Google privacy policy? We're using Google Cloud, right? The Google Cloud project belongs to the Apache Beam community, so the data is never under Google's responsibility. (and we should never let Google own this data in any way - only community members can have access to it)

In short, I don't think the Google privacy policy applies here

Reply:
We were following the DartPad way – https://dart.dev/tools/dartpad/privacy
Do you think we should change the privacy policy?

Reply:
Pablo is correct. We shouldn't apply the Google policy here but the more specific GCP policy available here: https://cloud.google.com/terms/cloud-privacy-notice.